distributed/dashboard/tests/test_scheduler_bokeh.py

```diff
@@ -430,6 +430,19 @@ def metric(worker):
     assert s.workers[a.address].metrics["metric"] == -999
 
 
+@gen_cluster(client=True)
+async def test_WorkerTable_with_memory_limit_as_0(c, s, a, b):
+    s.workers[a.address].memory_limit = 0
+    s.workers[b.address].memory_limit = 0
```

Comment:
I think that something like this should work instead, and probably relies only on public API rather than reaching in

```suggestion
@gen_cluster(client=True, worker_kwargs={"memory_limit": 0})
async def test_WorkerTable_with_memory_limit_as_0(c, s, a, b):
```

Reply:
+1 from me otherwise

Reply:
Oh nice! I didn't look closely at what was going on in the test setup :facepalm: